CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - ConfigGroup V2 (https://github.com/pulumi/pulumi-kubernetes/pull/2844)
 - ConfigFile V2 (https://github.com/pulumi/pulumi-kubernetes/pull/2862)
 - Bugfix for ambiguous kinds (https://github.com/pulumi/pulumi-kubernetes/pull/2889)
+- [yaml/v2] Improved resource prefix (https://github.com/pulumi/pulumi-kubernetes/pull/2889)
```

Comment:
Since this hasn't shipped yet, I don't think we need a changelog entry since it can just be rolled under the ConfigGroup V2 release entry.